js/src/forum/addSettingsItems.js

```diff
@@ -0,0 +1,139 @@
+import { extend } from "flarum/extend";
```

Comment:
I think single quotes should be used for strings, here and in the rest of the changes. I don't think there's any automated style check in this repository, but other files follow this convention.

Reply:
I'll change them over. Darn auto-format!